package.json

```
@@ -1,11 +1,16 @@
 {
   "name": "@mochajs/json-file-reporter",
   "version": "1.2.0",
+  "author": "Munieru",
```

Please use object style to include URL.

```suggestion
  "author": {
    "name": "Munieru",
    "url": "https://github.com/munierujp"
  },
```

cf. https://docs.npmjs.com/files/package.json#people-fields-author-contributors